scripts/GameWorld.gd

```diff
@@ -4,7 +4,3 @@ extends Spatial
 # Declare member variables here. Examples:
```

Comment:
Unclear to me what GameWorld.gd is...And how you choose what gets instantiated here and not elsewhere

Reply:
Oh, this comment was autogenerated by godot.

These should be renamed, but basically:

* GameWorld: contains sun, character, and Pie HUD canvas element
  - Contains a World, and probably any scene things like houses and towns
    - Contains an Island, which contains the world mesh and all the trees